vignettes/site_only/db_connections.Rmd

```diff
@@ -0,0 +1,204 @@
+---
+title: "Database Connections"
```

Comment:
@mmastand thanks for this vignette! This will make life so much easier both for us and the team. Appreciate it!